### ISSUE TITLE
brave shield global setting is not working when brave is closed or restarted

### ISSUE THREAD

**Issue by norbertc** [brave/brave-browser#6708]
## Description 
Brave Shield global setting does not take effect after the browser is restarted

## Steps to Reproduce

   1.  Disable blocking of cross-site trackers and HTTPS everywhere in Brave Shield global settings
   2.  At first, it seems the global setting is working after visiting some site.
   3.  Close Brave Browser and open it again
   4. Visit a random site and the brave shield for that site has enabled blocking of cross-site tracker and enabled HTTPS although disabled in Brave Shield global settings.

## Actual result:

Visit a random site and the brave shield for that site has enabled blocking of cross-site tracker and enabled HTTPS but still disabled in Brave Shield global settings.

## Expected result:
It should respect the Brave shield global settings on which blocking of cross-site tracker and HTTP everywhere should be off.

## Reproduces how often: 

Easily reproduced. A one or two times restart of Brave browser can easily reproduce the issue.

## Brave version (brave://version info)

Tested on 0.71.107 Chromium: 78.0.3904.70 (Official Build) beta (64-bit). Installed using the standalone installer.

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel? Not tested
- Can you reproduce this issue with the nightly channel? Not tested

**Comment by rebron** [comment]
@cezaraugusto Can you take a look at this one?

**Comment by bsclifton** [comment]
@norbertc I can't reproduce on the latest release channel build (1.0.1) - can you please update and try again?

**Comment by norbertc** [comment]
@bsclifton I was able to reproduce it on latest beta (1.2.34). these are the steps I did:

1. disable block cross-site trackers, upgrade connection to https and block scripts in shields settings from brave://settings/shield (I also toggled it on and off)
2. close the browser
3. visit a random site and the shield for that site is set as up. brave logo is not grayed out and cross-site tracker and https anywhere is enabled. but if you look at brave://settings/shield, they are still set to disabled.
